glitch/404.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Google tag (gtag.js) -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-1752YPLP7H"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "G-1752YPLP7H");
+    </script>
+
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link type="text/css" rel="stylesheet" href="css/404.css" />
+    <title>Glitch | tsParticles 404</title>
+    <link rel="icon" type="image/x-icon" href="../images/tsParticles-64.png" />
+  </head>
+
+  <body>
+    <div id="glitch">404</div>
+
+    <script src="https://cdn.jsdelivr.net/npm/tsparticles@2.9.3/tsparticles.bundle.min.js"></script>
```

Comment:
Can you please use @2 version, so it takes the latest v2 instead of a specific one? Also on the script below.